### PR TITLE
adding blog site

### PR DESCRIPTION
**Pull request by noobmaster432**

### CODE REVIEW

File: src/blocks.js [hunk 1]
```diff
@@ -1461,6 +1461,13 @@
     "block_icon": "./blocks/yashraj2003e/icon.png",
     "author_name": "Yashraj",
     "author_github_url": "https://github.com/yashraj2003e/"
+  },
```
Comment (sophiabrandt) [inline src/blocks.js:1464]: Please see the guide and revert your changes to this file:

> Please keep src/blocks.js as it is. If you follow step 10 above, a script will add your art automatically after your PR is merged.

File: src/blocks/noobmaster432/css/style.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
+* {
+  margin: 0;
+  padding: 0; 
+}
+
+.navigation {
+  margin-top: 25px;
+  font-family: var(--font1);
+  /* height: 74px;  */
```
Comment (sophiabrandt) [inline src/blocks/noobmaster432/css/style.css:9]: Please remove outdated code

File: src/blocks/noobmaster432/css/style.css [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+.home-article-content {
+  align-self: center;
+  padding: 25px;
+}
+
+.home-article-content a {
+  text-decoration: none;
+  color: black;
+}
+/* .home-articles{} */
```
Comment (sophiabrandt) [inline src/blocks/noobmaster432/css/style.css:126]: Same here.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
1. Please provide the licenses to the images and icons you used, so that we can be sure that we don't violate copyright.

2. Please fill out the request template instead of deleting it.

**Comment by MattCSmith** [comment]
Closing due to unresolved change request. Feel free to resolve and reopen.

**Pull request closed without merging**